lib/headquarters/api/endpoints.rb

```diff
@@ -0,0 +1,12 @@
+module Headquarters
+  class API
+    module Endpoints
```

Comment:
there are no methods being mixed in

with your solution, we would do `API::MEMBERS`, which doesn't even make sense
with the current one, we do `API::Endpoints::MEMBERS`